### DISCUSSION TITLE
添加了一个小功能

### DISCUSSION THREAD

**Discussion by whoistiger** [lsky-org/lsky-pro#468] [分享]
验证图片访问来源，直接访问图片会跳转到展示界面。

https://tutu.to/ttt_Mwysgb.jpg -> https://tutu.to/show/ttt_Mwysgb-jpg

使用`img`标签或者 Markdown 展示都正常显示。

[![](https://tutu.to/ttt_Mwysgb.jpg)](https://tutu.to/ttt_Mwysgb.jpg)

<a href="https://tutu.to/ttt_aZixwo.jpg"><img src="https://tutu.to/ttt_aZixwo.jpg"></a>

**Comment by heiyuan0801** [comment]
牛啊有没有教程
